static/src/javascripts-legacy/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -187,6 +189,10 @@ define([
             blockEngagementBanner: options.blockEngagementBanner || false,
             engagementBannerParams: options.engagementBannerParams || {},
             isOutbrainCompliant: options.isOutbrainCompliant || false,
+
+            // Set useLocalViewLog to true if only the views for the respective test
+            // should be used to determine variant viewability
+            useLocalViewLog: options.useLocalViewLog || false,
```

Comment:
yep, i think it makes sense there as the test is what's used when counting the view log, not the variant